translator/util/ec2util/ec2util.go

```diff
@@ -43,6 +46,14 @@ func initEC2UtilSingleton() (newInstance *ec2Util) {
 		return
 	}
 	md := ec2metadata.New(ses)
+	for i := 0; i < allowedRetries; i++ {
+		if md.Available() {
+			continue
```

Comment:
I think `break` makes more sense than `continue`.
If it is available on the first loop, we do not need to loop anymore.

Reply:
Oops that's my bad. I meant to use `break` here.